### PR TITLE
[12.0][MIG] stock_orderpoint_purchase_link

### PR DESCRIPTION
**Pull request by kittiu**
First submission.

### CODE REVIEW

**Comment by rousseldenis** [comment]
Depends on : 
* [ ] #550

**Comment by rousseldenis** [comment]
@kittiu Thanks for migrating.

Could you please rename your commit for migration like '[12.0][MIG] stock_orderpoint_purchase_link' ?

**Comment by MiquelRForgeFlow** [comment]
@kittiu  please, rebase this PR :)

**Comment by kittiu** [comment]
> @kittiu please, rebase this PR :)

@mreficent , done it. (but I am quite confused with rebase thing, please suggest if I make it not clean enough).

**Comment by MiquelRForgeFlow** [comment]
Good. Now the CIs should be greenified :)

**Comment by LoisRForgeFlow** [comment]
This is not correctly rebased, when we say rebase we meant that you have to do the folloing actions:
```bash
git checkout your-pr-branch
git fetch oca-remote 11.0
git rebase oca-remote/11.0 
``` 

In this case it will not work as the history here is quite messed up. You can, for simplicity and to avoid more complex git commands (btw, I suggest you to take some time to understand better the git capabilities, with a online tutorial for example):

1. rename this branch:
`git branch -m backup-branch`

2. Copy the module folder to other location (manually).
3. Re-do the migration process as you already did in the first place. (follow the wiki). This will recreate the branch of this PR (12.0-mig-stock_orderpoint_purchase_link)
4. Paste (manually) the module migrated you copied before.
5. Commit the changes and force push to this PR's branch
```
git add .
git commit -m "[12.0][MIG] stock_orderpoint_purchase_link"
git push -f your-remote 12.0-mig-stock_orderpoint_purchase_link -u
```

**Comment by kittiu** [comment]
@lreficent thank you for your time explaining. I will do it again.

File: stock_orderpoint_purchase_link/tests/test_stock_orderpoint_purchase_link.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+            'name': 'Test',
+            'location_id': self.warehouse.view_location_id.id,
+        })
+        route_buy = self.env.ref('purchase_stock.route_warehouse0_buy').id
+        route_test = self.route_obj.create({
+            'name': 'Stock to Test',
+        }).id
+        self.rule_obj.create({
+            'name': 'Stock to Test',
+            'action': 'move',
```
Comment (LoisRForgeFlow) [inline stock_orderpoint_purchase_link/tests/test_stock_orderpoint_purchase_link.py]: change to `pull`

Comment (kittiu) [reply stock_orderpoint_purchase_link/tests/test_stock_orderpoint_purchase_link.py]: OK, done.
Thank you!

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM code review

**Review by rousseldenis — APPROVED** [review]
Code review

**Pull request merged**